Leanplum-tvOS-SDK-source.podspec

```diff
@@ -0,0 +1,32 @@
+#
+# Be sure to run `pod lib lint Leanplum-SDK.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name = 'Leanplum-tvOS-SDK-source'
+  s.version = '2.0.3-SNAPSHOT'
+  s.summary = 'Mobile Marketing Platform. Integrated. ROI Engine.'
+  s.description = <<-DESC
+Leanplum’s integrated solution delivers meaningful engagement across messaging and the in-app
+experience. We offer Messaging, Automation, App Editing, Personalization, A/B Testing, and
+Analytics.
+    DESC
+  s.homepage = 'https://www.leanplum.com'
+  s.license = { :type => 'Commercial', :text => 'See https://www.leanplum.com/tos' }
+  s.author = { 'Leanplum' => 'support@leanplum.com' }
+  s.social_media_url = 'https://twitter.com/leanplum'
+  s.requires_arc = true
+  s.source = { :git => 'https://github.com/Leanplum/Leanplum-SDK-Internal.git', :tag => s.version.to_s }
```

Comment:
This is 404. We should link to https://github.com/Leanplum/Leanplum-tvOS-SDK-internal.